test/mpi/util/mtest_f90.f90

```diff
@@ -0,0 +1,150 @@
+! This file created from f77/util/mtestf.f with f77tof90
```

Comment:
Should we remove this line about how the file was created? How is this version modified from the old generated version?